docs/user/BuildLocally.md

```diff
@@ -12,6 +12,21 @@ cd OpenROAD-flow-scripts
 sudo ./setup.sh
 ```
 
+## Using Bazel to build OpenROAD and run the ORFS flow
+
+Long story short: OpenROAD will eventually switch to using Bazel for downloading dependencies and building OpenROAD for all the reasons that the DependencyInstaller.sh and cmake are hard to support and brittle across platforms.
+
+Currently the simplest way to build OpenROAD and run ORFS is to run one test, which will download all OpenROAD dependencies and build OpenROAD in the exec configuration:
+
+``` shell
+cd tools/OpenROAD
+bazelisk test src/drt/...
```

Comment:
why test drt?

Reply:
there is no explicit way in bazel to build an exec version of a binary, it is built because the exec binary is used by another rule